streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -92,7 +93,7 @@
 
     /**
      * Materialize this stream to a topic, also creates a new instance of {@link KStream} from the topic
-     * using default serializers and deserializers.
+     * using default serializers and deserializers and producer's {@link org.apache.kafka.streams.kstream.internals.WindowedStreamPartitioner}.
```

Comment:
I think we only use a wrapped WindowedStreamPartitioner if keySerializer is instance of WindowedSerializer?